### PR TITLE
*: refine log of checker

### PR DESCRIPTION
**Pull request by GregoryIan**
### What problem does this PR solve? 

Eliminate some doubtful logs in checker

### What is changed and how it works?

add more information into error and log

### Check List 

Tests 

 - No code

### CODE REVIEW

File: checker/checker.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func (c *Checker) Init() (err error) {
 		}
 		instance.sourceDB, err = dbutil.OpenDB(*instance.sourceDBinfo)
 		if err != nil {
-			return errors.Trace(err)
+			return errors.Annotatef(err, "failed to open source DSN %s:***@%s:%d", instance.cfg.From.User, instance.cfg.From.Host, instance.cfg.From.Port)
```
Comment (amyangfei) [inline checker/checker.go]: shall we differ use and no password scenario, it is more explicit. (although we can detect whether password is used in DSN from the `err`)
if password is not empty, annotate error with `user:***@host:port` ,
without password, annotate error with `user@host:port`

Comment (GregoryIan) [reply checker/checker.go]: I think no need

Comment (amyangfei) [reply checker/checker.go]: ok

**Pull request merged**